docs/src/main/sphinx/sql/show-catalogs.rst

```diff
@@ -13,4 +13,10 @@ Description
 -----------
 
 List the available catalogs.
-The ``LIKE`` clause can be used to restrict the list of catalog names.
+
+:ref:`Specify a pattern <like_operator>` in the optional ``LIKE`` clause to
+filter the results to the desired subset. For example, the following query
+allows you to find catalogs where the first character in a value is specified.
+Query results show catalogs that begin with ``t``::
```

Comment:
```suggestion
filter the results to the desired subset. For example, the following query
allows you to find catalogs whose names begin with ``t``::
```

Similar in other places too? Existing wording seems repetitive (first we describe what query does in very abstract terms and then follow with the concrete meaning).

Reply:
I like that .. want to adapt to that @Jessie212 ?

Reply:
Great suggestion, @hashhar. Applied.